### ISSUE TITLE
should username be generated from `emailClaim` or `userClaim@clusterName` from config files?

### ISSUE THREAD

**Issue by paulczar** [heptiolabs/gangway#86]
According to the Documentation in the deployment manifests [config-map.yaml](https://github.com/heptiolabs/gangway/blob/master/docs/yaml/02-config.yaml#L68-L71) the config setting emailClaim should be used to `name the "user" part of the config`.

However recent changes [1](https://github.com/heptiolabs/gangway/pull/79/files) [2](https://github.com/heptiolabs/gangway/pull/75/files) has it being built by `userClaim`@`clusterName` from the config. 

What is the intention?  If the recent changes are the way to move forward then we can update the config and pull the emailClaim out completely,  otherwise we should revert back to using the `emailClaim`.

**Comment by stevesloka** [comment]
The user in the `context` needs to match the user in the `user` section. We need a way to make the user unique since we (soon) will have the ability to support multiple clusters. 

To stay consistent, we should probably change it up to be `Email@Cluster`. Does that make sense?

It would be difficult to keep the same user for multiple contexted and be manged by Gangway (i.e. we'd need a local tool to parse through your on-disk kubeconfig).

**Comment by paulczar** [comment]
yup that makes sense to me.   I think `userClaim`@`clusterName` as it's written makes sense, so we should deprecate `email` as a setting in the config as it wouldn't be used any more.   I think `emailClaim`@`clusterName` would be visually weird and not really provide anything more than the former.

so I can close up my PRs to fix it back to `emailClaim` and PR doc fixes and a deprecation.

**Comment by paulczar** [comment]
I believe I've cleared this up in #87.    If emailClaim is present in config it will use it and print out a deprecation message, otherwise it'll save `usernameClaim@clusterName` as the email field and the templates will use that.
